src/sql/platform/dialog/dialogPane.ts

```diff
@@ -93,12 +93,12 @@ export class DialogPane extends Disposable implements IThemable {
 	}
 
 	private getTabDimension(): DOM.Dimension {
-		return new DOM.Dimension(DOM.getContentWidth(this._body), DOM.getContentHeight(this._body));
+		return new DOM.Dimension(DOM.getContentWidth(this._body) - 5, DOM.getContentHeight(this._body) - 5);
```

Comment:
What's the `- 5` for, to account for the scrollbars? Comments might be helpful here

Reply:
yes I'm trying to fix the scrollbar issue but still not be able to. this seems to help with fitting the tab content inside the tab 

---
In reply to: [198311889](https://github.com/Microsoft/sqlopsstudio/pull/1737#discussion_r198311889) [](ancestors = 198311889)